packages/core/src/__tests__/ceramic-api.test.ts

```diff
@@ -41,7 +41,7 @@ describe('Ceramic API', () => {
 
   const MODEL_DEFINITION: ModelDefinition = {
     name: 'MyModel',
-    accountRelation: ModelAccountRelation.LIST,
+    accountRelation: { type: 'list' },
```

Comment:
Are you sure that does not affect existing models?
I guess, there are no existing models, but still.

Reply:
I expect it will affect existing models, this should be considered a breaking change.